HousingRepairsOnlineApi.Tests/Domain/SorConfigurationExtensionsTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Linq;
```

Comment:
It seems that `using System.Linq;` is an unused import.

Reply:
Removed